include/rnp/rnp_def.h

```diff
@@ -77,6 +77,7 @@ enum {
     RNP_ERROR_NOT_SUPPORTED,
     RNP_ERROR_OUT_OF_MEMORY,
     RNP_ERROR_SHORT_BUFFER,
+    RNP_ERROR_NULL_POINTER,
```

Comment:
You may want to just use `RNP_ERROR_BAD_PARAMETERS` (which IMO would be better named `RNP_ERROR_INVALID_ARGS` or something, but maybe that's just the US)

Reply:
Yeah could go either way on this, I kind of like `NULL_POINTER` since it explicitly calls out the issue vs `BAD_PARAMS` which could be referring to anything. But no strong feelings on it.

Reply:
Yeah I think I agree with you after looking again, there's a useful distinction here.